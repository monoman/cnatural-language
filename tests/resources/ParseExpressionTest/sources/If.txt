() => {
	if (a > b) {
		b += 1;
		return b;
	} else
		return a;
}