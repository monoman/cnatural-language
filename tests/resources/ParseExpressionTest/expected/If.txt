Lambda( => {
if ((Name(a) GreaterThan Name(b)))
{
Assign(Name(b) Add Literal(DecimalInteger, [1]));
return(Name(b));
}
else
return(Name(a));
}
)